reactDashboard/src/components/GeoHeatMap/GeoHeatMap.css

```diff
@@ -18,20 +19,28 @@
   display: none;
 }
 
-#map, #regionmap {
+#map,
+#regionmap {
   height: 100%;
   width: 100%;
+  margin: 0 5px;
+}
+
+.darkmode #map,
+.darkmode #regionmap {
+  border: 1px solid darkgray;
```

Comment:
Why is this block darkmode only ? ie why not lightmode version too?

Reply:
Defines the edge of the map component.

Reply:
Worth commenting on this for future maintainers

Reply:
There is a light mode block above it - not sure if this was added after the comment was made

Reply:
I thought we discussed adding the border to lightmode and only overriding the colour here?

Reply:
added border for light mode

Reply:
added border for lightmode